<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Renu" failures="0" tests="3" name="testcases.LogIN" time="14.227" errors="1" timestamp="2022-03-14T12:36:34 IST" skipped="2">
  <testcase classname="testcases.LogIN" name="loginTest" time="0.035">
    <skipped/>
  </testcase> <!-- loginTest -->
  <system-out/>
  <testcase classname="testcases.LogIN" name="loginTest" time="13.634">
    <error message="Cannot invoke &quot;org.testng.ITestResult.getTestContext()&quot; because the return value of &quot;org.testng.Reporter.getCurrentTestResult()&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.testng.ITestResult.getTestContext()" because the return value of "org.testng.Reporter.getCurrentTestResult()" is null
at com.zoho.listener.ZohoeventListener.getSession(ZohoeventListener.java:45)
at com.zoho.listener.ZohoeventListener.beforeFindBy(ZohoeventListener.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.openqa.selenium.support.events.EventFiringWebDriver.lambda$new$0(EventFiringWebDriver.java:85)
at jdk.proxy2/jdk.proxy2.$Proxy14.beforeFindBy(Unknown Source)
at org.openqa.selenium.support.events.EventFiringWebDriver.findElement(EventFiringWebDriver.java:194)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:182)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:179)
at org.openqa.selenium.support.ui.FluentWait.lambda$checkConditionInLoop$2(FluentWait.java:233)
at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1764)
at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1756)
at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295)
at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
]]>
    </error>
  </testcase> <!-- loginTest -->
  <system-out/>
  <testcase classname="testcases.LogIN" name="loginTest" time="0.558">
    <skipped/>
  </testcase> <!-- loginTest -->
  <system-out/>
</testsuite> <!-- testcases.LogIN -->
